components/Header.vue

```diff
@@ -49,6 +49,11 @@ export default {
         : { name: "cart" }
     },
   },
+  methods: {
+    openSidebarCart() {
+      return this.$store.dispatch("cart/openSidebarCart", true)
```

Comment:
Can't `mapActions` be used here too?

Reply:
Not sure about this one...how you'll do it?

Reply:
```
diff --git a/components/Header.vue b/components/Header.vue
index 5d64032..3f6f51c 100644
--- a/components/Header.vue
+++ b/components/Header.vue
@@ -11,7 +11,7 @@
         .navbar-item
           .field
             p.control
-              a.button.is-light(@click="openSidebarCart()")
+              a.button.is-light(@click="openSidebarCart(true)")
                 span.icon.cartitem
                   .cartcount(v-if="total > 0") {{ total }}
                   i.fa.fa-shopping-cart
@@ -26,7 +26,7 @@
 <script>
 import { createNamespacedHelpers, mapGetters } from "vuex"
 
-const { mapGetters: cartGetters } = createNamespacedHelpers("cart")
+const { mapGetters: cartGetters, mapActions } = createNamespacedHelpers("cart")
 
 export default {
   name: "AppHeader",
@@ -50,9 +50,7 @@ export default {
     },
   },
   methods: {
-    openSidebarCart() {
-      return this.$store.dispatch("cart/openSidebarCart", true)
-    },
+    ...mapActions(["openSidebarCart"]),
   },
 }
 </script>

```

But yeah, maybe passing true in a vue template is not ideal

Reply:
It's not ideal and tried that a few minutes ago and (don't know why) got an awful error like "Maximum call stack size exceeded"...

Reply:
It's because you have probably put mapActions into `computed` and not `methods`.

Reply:
You're right! Solved on the last commit!